src/services/addAddresses.ts

```diff
@@ -0,0 +1,52 @@
+import { Customer } from '../types/user/customer';
+import { FormData } from '../types/user/formData';
+
+export const addAddresses = async (data: FormData, customer: Customer, userToken: string) => {
+  console.log(customer);
+
+  const isShippingDefaultAddress = data.isShippingDefaultAddress;
+  const isBillingDefaultAddress = data.isBillingDefaultAddress;
+  const isBillingSameAsShipping = data.isBillingSameAsShipping;
```

Comment:
В целом, если наш объект data содержит в себе эти поля, мы можем деструктурировать объект, и достать оттуда сразу эти переменные
```suggestion
  const { isShippingDefaultAddress, isBillingDefaultAddress, isBillingSameAsShipping } = data;
```

Reply:
Исправил. Одна строчка, лучше трех ))